backend/src/main/java/de/kathahuth/gymast/service/CategoryService.java

```diff
@@ -0,0 +1,28 @@
+package de.kathahuth.gymast.service;
+
+import de.kathahuth.gymast.db.CategoryMongoDb;
+import de.kathahuth.gymast.model.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CategoryService {
+
+    private final CategoryMongoDb categoryMongoDb;
```

Comment:
`PagingAndSorting` `interface` already hides your database implementation, so there is no need to reflect the specific implementation within the namespace outside of the  `interface`: `CategoryMongoDb -> CategoryDb`